sway/ipc-json.c

```diff
@@ -30,6 +30,20 @@ static const char *ipc_json_layout_description(enum sway_container_layout l) {
 	return "none";
 }
 
+static const char *ipc_json_orientation_description(enum sway_container_layout l) {
+	switch (l) {
```

Comment:
This case statement is a bit too heavy for my liking, since it's just a return statement and there are many fall-throughs. Personally, I'd make it a ternary, though I think even an if block would be better.